src/main/java/com/avispl/symphony/dal/infrastructure/management/polycom/lens/PolyLensCommunicator.java

```diff
@@ -155,7 +156,7 @@ public void run() {
 				try {
 					TimeUnit.MILLISECONDS.sleep(500);
 				} catch (InterruptedException e) {
-					// Ignore for now
+					logger.info("Ignore for now");
```

Comment:
Can you please provide more meaningful info message? So that we can see that the interrupted exception was caught, but no extra action is required